### ISSUE TITLE
Feature Suggestion: Add Custom Difficulty Button for Levels

### ISSUE THREAD

**Issue by rubiidev18alt** [Uproxide/more-difficulties#7]
## Description

This feature suggestion proposes adding a button to the level menu in Geometry Dash (when using the More Difficulties Mod) that allows players to assign custom difficulties to specific levels. This would enhance player customization and offer more granular control over the difficulty experience.

**Implementation Details:**

* A new button labeled "Add Custom Difficulty" could be placed alongside the existing "Copy" button in the level menu.
* Clicking this button would open a dialogue box where players can:
    * Upload a difficulty face using an upload button
    * Name that difficulty
    * Save the difficulty
After you upload that difficulty, you can:
    * Select the desired custom difficulty from a dropdown menu for any level
* Once selected and saved, the chosen difficulty would be displayed next to the level name in the level selection screen, along with the original difficulty (if applicable).

**Benefits:**

* Players can personalize their experience by assigning difficulties that better reflect the actual challenge of a level. 
* This feature caters to players of all skill levels, allowing beginners to practice demon-like sections in easier difficulties, while veterans can push themselves with custom "Extreme" versions of regular levels.
* Fosters a more creative and community-driven approach to difficulty settings.

**Considerations:**

* The implementation should ensure compatibility with existing More Difficulties Mod functionalities.
* Visual clarity is important. The custom difficulty indicator should be easily distinguishable from the original difficulty.

We believe this feature would be a valuable addition to the More Difficulties Mod, offering players greater control over their Geometry Dash experience.

**Comment by DevonMKatYT** [comment]
YES I WANTED TO MAKE A MOD WHERE YOU COULD DO THIS! UPROXIDE PLEEEAAASE

**Comment by DevonMKatYT** [comment]
The only thing I would add is that a setting to display the original difficulty or not. And possibly a deletion and sharing custom difficulties?
